src/main/java/nextstep/qna/domain/Answer.java

```diff
@@ -64,8 +65,17 @@ public NsUser getWriter() {
         return writer;
     }
 
-    public String getContents() {
-        return contents;
+
+    public DeleteHistory delete(NsUser loginUser) throws CannotDeleteException{
+        verifyIfErasable(loginUser);
+        setDeleted(true);
```

Comment:
```suggestion
        this.deleted = true;
```
setter 이름을 가지는 메서드에 대한 거부감이 있는데...
메서드를 분리한다면 다른 이름을 사용하는 것은 어떨까?